python/pyspark/ml/tests.py

```diff
@@ -905,6 +906,43 @@ def test_linear_regression_summary(self):
         sameSummary = model.evaluate(df)
         self.assertAlmostEqual(sameSummary.explainedVariance, s.explainedVariance)
 
+    def test_linear_regression_summary(self):
```

Comment:
this method is the same name as the one above - should it perhaps be `test_glm_summary`?